html/static/script.js

```diff
@@ -161,3 +161,14 @@ function updateSample(element, uid) {
       alert('Error in updating the dictionary: ' + data.responseText);
     })
 }
+
+function changePage(element, page) {
+  var loc = location.href;
+  if (page === 0) page =1; 
+  if (loc.indexOf("?") === -1)
+    loc += "?page=" + page;
```

Comment:
This works until it doesn't :cat: 
I would suggest you to look at JavaScript's `URLSearchParams`. It helps to convert URL parameters to a dictionary, so you don't have to play with cutting and gluing strings. It works well when you have only one parameter there, but might get messy when you start having more. Try this and see if this would help:
```
let urlParams = Object.fromEntries(new URLSearchParams(window.location.search));
urlParams["page"] = page;
location.search = "?" + new URLSearchParams(urlParams).toString();
```